packages/store/src/IStore.sol

```diff
@@ -54,6 +54,13 @@ interface IStoreWrite {
   function deleteRecord(bytes32 table, bytes32[] memory key) external;
 }
 
+interface IStoreEphemeral {
+  event StoreEphemeralSetRecord(bytes32 table, bytes32[] key, bytes data);
+
+  // Emit the ephemeral event without modifying storage
+  function setEphemeralRecord(bytes32 table, bytes32[] calldata key, bytes calldata data) external;
```

Comment:
Similar to above, if we think we'll have other operations (e.g. delete), then I think this is fine as is.

If we're only planning to have set (i.e. `emit StoreEphemeralRecord`), then I think it'd be clearer to have this as `emitEphemeralRecord` to indicate we're not storing anything.